services/conference/src/scripts/stores/participants/ParticipantBase.ts

```diff
@@ -22,6 +22,17 @@ export class ParticipantBase extends MapObject implements Store<IParticipantBase
     console.log(storage === localStorage ? 'Save to localStorage' : 'Save to sessionStorage')
     storage.setItem('localParticipantInformation', JSON.stringify(this.information))
   }
+  loadInformationToStorage() {
+    let storage = localStorage
+    if (sessionStorage.getItem('localParticipantInformation')) {
+      storage = sessionStorage
+    }
+    console.log(storage === localStorage ? 'Load from localStorage' : 'Load from sessionStorage')
+    const infoInStr = storage.getItem('localParticipantInformation')
+    if (infoInStr) {
+      this.information = JSON.parse(infoInStr)
```

Comment:
This will clear all the registered observers on this.information.name ... .
Object.assign(this.information, JSON.parse(infoInStr)) would be better.